diffpy/pdfmorph/pdfmorphapp.py

```diff
@@ -227,9 +239,130 @@ def custom_error(self, msg):
 def main():
     parser = create_option_parser()
     (opts, pargs) = parser.parse_args()
+    if opts.sequence:
+        multiple_morphs(parser, opts, pargs, stdout_flag=True)
+    else:
+        single_morph(parser, opts, pargs, stdout_flag=True)
+
+
+def multiple_morphs(parser, opts, pargs, stdout_flag):
+    # Custom error messages since usage is distinct when --sequence tag is applied
+    if len(pargs) < 2:
```

Comment:
This seems a bit brittle and unmaintainable to me since we may add or remove parser options in the future (see my comments above).  Can we find a more robust way to do this?

Reply:
Since most parser options are shared by single_morph and multiple_morphs, changing most parser options would only require changes in the single_morph function. Options that are not shared between them (plotting and saving since different items are plotted/saved) are the only ones kept separate right now.